e2e/cypress/integration/03-admin/03-state-admin-portal.cy.js

```diff
@@ -23,10 +23,14 @@ const verifyRole = (name, role) => {
     });
 };
 
-describe('tests state admin portal', () => {
-  it('tests state admin portal', { tags: ['@state', '@admin'] }, async () => {
-    // Request access on No Role
+describe('tests state admin portal', async () => {
+  before(() => {
     cy.task('db:resetnorole');
+  });
+
+  it('tests state admin portal', { tags: ['@state', '@admin'] }, () => {
+    // Request access on No Role
+    // cy.task('db:resetnorole');
```

Comment:
why aren't you resetting it any more?

Reply:
I remember this test giving me some trouble. I think It's because the async and cy.task was messing with the code instrumentation. I'll look back into it and see If I can get it working with resetnorole again

Reply:
Update: Added `cy.task('db:resetnorole')` but left out the `async ` in line 31 as that is what caused the test failure.
Is that ok @thetif?